lib/rbs/prototype/rb.rb

```diff
@@ -411,16 +411,22 @@ def literal_to_type(node)
         when :HASH
           list = node.children[0]
           if list
-            children = list.children.compact
+            children = list.children
+            children.pop
```

Comment:
In RubyVM::AST, all list has trailing nil. It removed the nil by `compact` previously. But `compact` removes a nil for hash spread unexpectedly. So I changed `compact` to `pop`.